README.md

```diff
@@ -76,6 +79,9 @@ libdebug offers many advanced features. Take a look at this script doing magic w
 
 ```python
 from libdebug import debugger
+from libdebug import libcontext
```

Comment:
from libdebug import debugger, libcontext

Reply:
Addressed in [9e4a7f8](https://github.com/libdebug/libdebug/pull/129/commits/9e4a7f831a709f7ca168593df7deb81678b86ed5)